api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/repository/v1/SdcDistrictCollectionRepository.java

```diff
@@ -2,12 +2,22 @@
 
 import ca.bc.gov.educ.studentdatacollection.api.model.v1.SdcDistrictCollectionEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 import java.util.Optional;
 import java.util.UUID;
 
 @Repository
 public interface SdcDistrictCollectionRepository extends JpaRepository<SdcDistrictCollectionEntity, UUID> {
+
+  @Query(value="""
```

Comment:
@eckermania can you use findByDistrictIDAndSdcDistrictCollectionStatusCodeNotIgnoreCase instead? We decided to use status instead of collection dates

Reply:
Definitely - I'll make this change